interp/interp.go

```diff
@@ -528,6 +549,62 @@ func (interp *Interpreter) Use(values Exports) {
 			interp.binPkg[k][s] = sym
 		}
 	}
+	if _, ok := values["fmt"]; ok {
+		fixStdio(interp)
+	}
```

Comment:
what about the "log" package? what about any imported code that has its own printing functions/methods?
Ah, I see you're also hacking the log symbols L581. But are we only relying on the presence of "fmt" as a detection (L552) because you think any code that deals with input/output will always import "fmt" anyway?

Reply:
The call to `fixStdio()` needs to be done just after a `i.Use(stdlib.Symbols)`, and would be irrelevant for other arguments. Testing for a `"fmt"` package path achieves that. The only assumption here is that `"fmt"` is part of `"stdlib"`. I'm adding a comment to clarify that.